packages/lesswrong/components/ea-forum/GlobalButtonBurst.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useEffect, useState } from "react";
+import ReactDOM from "react-dom";
+import { DatabasePublicSetting } from "@/lib/publicSettings";
+
+export const buttonBurstSetting = new DatabasePublicSetting<boolean>("buttonBurst.enabled", false);
+const buttonBurstImage = new DatabasePublicSetting<string>("buttonBurst.image", "https://res.cloudinary.com/cea/image/upload/w_256,h_256,q_40,f_auto,dpr_1/v1711484824/bulby.png");
```

Comment:
This is the one where he's peeking out from the side, we can create a better image before turning it on

<img src="https://res.cloudinary.com/cea/image/upload/v1711484824/bulby.png" />